### ISSUE TITLE
SCSS dependencies

### ISSUE THREAD

**Issue by pmarsceill** [electron/electron-compile#272]
When defining a path to load additional Sass/SCSS dependencies in `.compilerc` is it proper to use...

```
{
  "text/scss": {
    "paths": "./node_modules"
  }
}
```

or

```
{
  "text/scss": {
    "includePaths": "./node_modules"
  }
}
```

I've found a few conflicting PRs e.g., https://github.com/electron/electron-compilers/pull/78 and it is unclear to me how to load additional paths for dependencies.

**Comment by ErraticFox** [comment]
Did you ever figure this out? I tried both of these in my .compilerc and they both gave an error saying `paths` and `includePaths` was a invalid option.

**Comment by pmarsceill** [comment]
@ErraticFox no, I never did. After talking to some folks on the electron team, it seems like if I want to use SCSS dependencies from npm, I should probably just use vanilla Electron and manually setup my sass compiler using `node-sass` and setup the `includePaths` as I would with any other web project.

**Comment by cleanenergyjoy** [comment]
#85

**Comment by dfeinzeig** [comment]
@pmarsceill , @ErraticFox , any updates on this? trying to figure out best way to include sass within react components while using electron-compile.
